packetbeat/protos/http/http.go

```diff
@@ -701,6 +707,24 @@ func parseCookieValue(raw string) string {
 	return raw
 }
 
+func extractHostHeader(header string) (host string, port int) {
```

Comment:
This whole function can be replaced by `url.Hostname()` and `url.Port()` but those don't handles the case where an IPv6 address is not surrounded by brackets.